### DISCUSSION TITLE
Some questions about install.sh

### DISCUSSION THREAD

**Discussion by unknowndevQwQ** [coder/code-server#5643] [Q&A]
https://github.com/coder/code-server/blob/ba68656353edfd7a50ef4b5571322e2f0c1b653c/install.sh#L566

Does the comma after `sudo` seem redundant?

**Comment by lg-nineoneninethreenine** [comment]
It is not a comma, it's a full stop.

**Reply by unknowndevQwQ** [reply]
it's `sudo`, not `su`, sry

**Reply by lg-nineoneninethreenine** [reply]
Oh!  Sorry, yeah it does feel redundant.
